File: Restaurant-back/src/main/java/com/spring/restaurant/entites/Chef.java
package com.spring.restaurant.entites;

import jakarta.persistence.Entity;
import lombok.*;

@Entity
@NoArgsConstructor
@Getter
@Setter
public class Chef extends BaseEntity
{
	private String specialty;
	private String facebookLink;
	private String twitterLink;
	private String instagramLink;
}

File: Restaurant-front/src/model/Chef.ts
import {BaseEntity} from './BaseEntity';

export class Chef extends BaseEntity {
  specialty: string;
  facebookLink: string;
  twitterLink: string;
  instagramLink: string;


  constructor(chef: Partial<Chef>) {
    super(chef);
    if (chef) {
      this.specialty = chef.specialty;
      this.facebookLink = chef.facebookLink;
      this.twitterLink = chef.twitterLink;
      this.instagramLink = chef.instagramLink;
    }
  }
}
